src/BinSkim.Driver.FunctionalTests/BaselineTests.cs

```diff
@@ -85,8 +86,13 @@ private void RunRules(StringBuilder sb, string inputFileName)
         {
             string fileName = Path.GetFileName(inputFileName);
             string actualDirectory = Path.Combine(Path.GetDirectoryName(inputFileName), "Actual");
-            string expectedDirectory = Path.Combine(Path.GetDirectoryName(inputFileName), "Expected");
-
+            string expectedDirectory;
+            if(PlatformSpecificHelpers.RunningOnWindows()){

```

Comment:
More spacing issues. I will stop making this comment. We should consider running CodeFormatter against our code to get it into a uniform condition and scripting execution of this tool so that it's easy to maintain consistency.

Reply:
Agree that we should find a tool for this.  I went through the changes and tried to correct them whenever I found spacing issues.